internal/models/cli.go

```diff
@@ -147,7 +147,7 @@ type TokenBucket struct {
 	Size *int64 `json:"size"`
 }
 
-func GenerateConnectionFromOpts(opts *Options) (*nats.Conn, error) {
+func GenerateConnectionFromOpts(opts *Options, connectionName string) (*nats.Conn, error) {
```

Comment:
There is a `ConnectionName` in `Options`...

Reply:
dang....thanks 😅